src/http/api/account.ts

```diff
@@ -361,3 +370,30 @@ export function createGetAccountLikedPostsHandler(
         );
     };
 }
+
+/**
+ * Create a handler to handle a request for an account update
+ */
+export function createUpdateAccountHandler(
+    accountRepository: KnexAccountRepository,
```

Comment:
Just a reminder that we should be passing the service instead of the repository here, so that each layer only speaks to the layer below

Reply:
Fixed here : https://github.com/TryGhost/ActivityPub/pull/525/files